cognite/neat/rules/models/rules.py

```diff
@@ -553,29 +551,31 @@ def set_class_name_if_none(cls, value, values):
 
     @field_validator("parent_class", mode="before")
     @skip_field_validator("validators_to_skip")
-    def to_list_if_comma(cls, value, values):
+    def to_list_of_entities(cls, value, values):
         if isinstance(value, str):
             if value:
```

Comment:
No need to nest here
 ```suggestion
        if isinstance(value, str) and value: 
```